server/apps/publish/subscribers.py

```diff
@@ -18,6 +20,9 @@
 
 logger = logging.getLogger(__name__)
 
+subscriber_types = ['broadcast', 'digital', 'wire']
+SUBSCRIBER_TYPES = namedtuple('SUBSCRIBER_TYPES', ['BROADCAST', 'DIGITAL', 'WIRE'])(*subscriber_types)
```

Comment:
Just to be sure that I understand this
WIRE - publish to wire clients
DIGITAL - publish to digital clients, used mostly for takes/master files
BROADCAST - same as digital but in the case of takes will receive the take stories beside the take package, the only broadcast subscriber we know of currently, is PublicAPI
Are these assumptions correct?